addons/notifications/notifications.php

```diff
@@ -274,16 +274,18 @@ public function trash_question( $post_id, $_post ) {
 	 */
 	public function new_answer( $post_id, $_post ) {
 		$_question = get_post( $_post->post_parent );
-		ap_insert_notification(
-			array(
-				'user_id'  => $_question->post_author,
-				'actor'    => $_post->post_author,
-				'parent'   => $_post->post_parent,
-				'ref_id'   => $_post->ID,
-				'ref_type' => 'answer',
-				'verb'     => 'new_answer',
-			)
-		);
+		if ( $_question ) {
```

Comment:
@mebishalnapit Doing below will make code easy to read:

```
if ( !$_question ) {
    return;
}
```

Reply:
@rahularyan Updated.